lib/graphql/language/lexer.rl

```diff
@@ -49,7 +49,7 @@
   AMP =           '&';
 
   QUOTED_STRING = QUOTE STRING_CHAR* QUOTE;
-  BLOCK_STRING = BLOCK_QUOTE BLOCK_STRING_CHAR* BLOCK_QUOTE;
+  BLOCK_STRING = BLOCK_QUOTE QUOTE{0,2} BLOCK_STRING_CHAR* QUOTE{0,2} BLOCK_QUOTE;
```

Comment:
Hmm this allows `””””””””””` due to *, that’s out of spec. Will need to do something like `(QUOTE{1,2} ^QUOTE){,1}` to fix.